tests/integration_tests/tests/agentless_tests/deployment_update/test_deployment_update_misc.py

```diff
@@ -71,7 +73,9 @@ def update_deployment_wait_and_assert(dep,
         update_deployment_wait_and_assert(
             deployment, remodification_bp_path, 2, True, 'second')
 
-        executions = self.client.executions.list(is_descending=False)
+        executions = [ex for ex in
```

Comment:
Why not adding an assertion for the `upload_blueprint` execution too?

Reply:
how does it relate to what the test checks?